core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -31,8 +30,6 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   extraArgs: List[String]
 )
 
-final case class DeleteAppRequest(userInfo: UserInfo, cloudContext: CloudContext, appName: AppName, deleteDisk: Boolean)
```

Comment:
I don't feel too strongly here -- but all of the other `AppService` methods pass parameters directly and don't use a "request" object. So I just removed this to be consistent.